### PR TITLE
Feature/user confirmation

### PR DESCRIPTION
**Pull request by ivanfelfo**
Several changes in order to make users confirmable

### CODE REVIEW

File: bin/spring [hunk 1]
```diff
@@ -1,13 +1,15 @@
 #!/usr/bin/env ruby
```
Comment (mrodriguez90) [inline bin/spring:1]: This type of file should be ignored by rubocop.
Rubocop should only check code wrote by you or by others team members.

File: app/models/user.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
```
Comment (mrodriguez90) [inline app/models/user.rb:1]: Only needed when having string literals in the file.

**Review by mrodriguez90 — COMMENTED** [review]
PR should only contain changes related to the feature you're working on.
In the description, you should provide information about your implementation approach and the changes you’ve made.

File: app/models/user.rb (hunk 2 above)
Comment (ivanfelfo) [reply app/models/user.rb:1]: what do you mean by string literals?

File: app/controllers/v1/registrations_controller.rb [hunk 3]
```diff
@@ -1,12 +1,20 @@
...
     def create
-        user = User.new(email: params[:email], gender: params[:gender] ,password: params[:password])
-        if user.save!
-            render json: user.to_json(), status: :created
-        else
-            head(:unauthorized)
-        end
+      user = User.new(user_params)
+      if user.save!
+        render json: user.to_json, status: :created
```
[the remark this answers is not in the document]
Comment (ivanfelfo) [reply app/controllers/v1/registrations_controller.rb:8]: 👍

File: app/controllers/v1/registrations_controller.rb [hunk 4]
```diff
@@ -1,12 +1,20 @@
-class V1::RegistrationsController < ApplicationController
-    
-    
+# frozen_string_literal: true
+
+module V1
+  class RegistrationsController < ApplicationController
```
[the remark this answers is not in the document]
Comment (ivanfelfo) [reply app/controllers/v1/registrations_controller.rb:4]: 👍

File: app/controllers/v1/sessions_controller.rb [hunk 5]
```diff
@@ -1,11 +1,23 @@
-class V1::SessionsController < ApplicationController
+# frozen_string_literal: true
+
+module V1
+  class SessionsController < ApplicationController
```
[the remark this answers is not in the document]
Comment (ivanfelfo) [reply app/controllers/v1/sessions_controller.rb:4]: 👍

File: bin/spring (hunk 1 above)
Comment (ivanfelfo) [reply bin/spring:1]: 👍

File: app/controllers/v1/registrations_controller.rb (hunk 4 above)
[the remark this answers is not in the document]
Comment (mrodriguez90) [reply app/controllers/v1/registrations_controller.rb:4]: You can use the rails_api_base repo as a reference.

**Pull request closed without merging**